deployment/environments/prod/values.yaml

```diff
@@ -0,0 +1,28 @@
+networks:
+  - name: polygon-mainnet
+    host: polygon-mainnet.unode.klaos.laosfoundation.io/ 
+    debug: true
```

Comment:
I suggest removing the debug logs in prod. If you really want to keep them, I suggest removing them as soon as we publish v1.0

Reply:
+1 on that

Reply:
This also came to my mind. But since it is still not very stable I would keep debug in prod now, also. and yes, when we reach v1.0.0 we should remove debug logs. I agree.